pkg/commands/git_commands/git_command_builder_test.go

```diff
@@ -0,0 +1,56 @@
+package git_commands
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGitCommandBuilder(t *testing.T) {
+	scenarios := []struct {
+		input    string
+		expected string
+	}{
```

Comment:
If we're gonna leave the `if flag == ""` that @stefanhaller commented [in that other PR](https://github.com/jesseduffield/lazygit/pull/2646/files/76ed2047959b1cad5c8b9a8dbe9f509a1ffb1ce4#r1199569067) we might as well have a test for it.